pkg/kubelet/kubelet.go

```diff
@@ -2077,6 +2105,28 @@ func (kl *Kubelet) updateRuntimeUp() {
 		glog.Errorf("Container runtime not ready: %v", runtimeReady)
 		return
 	}
+
+	ci, err := kl.containerRuntime.GetRuntimeConfigInfo()
```

Comment:
yeah, this basically has us calling docker info every 5s if we don't cache the behavior in the shim.

i can see here why @feiskyer is suggesting putting the call in `Status` as a result so we can avoid making two grpc operations every 5s.  given this is the intended usage, i see a compelling argument for merging the information.

/cc @yujuhong @dchen1107 input appreciated.

Reply:
honestly, i dont think this information can change over the life of a kubelet.

calling the operation once, and caching the information should be sufficient imho.  operators that dynamically change the config will have had to drain the node already and most likely restart the kubelet.

so if we call this operation once, and cache the info, it doesnt need to merge into status.